src/modules/products/server/procedures.ts

```
@@ -1,6 +1,7 @@
 
 import { Categorise } from "@/payload-types";
 import { baseProcedure, createTRPCRouter } from "@/trpc/init";
+import { min } from "date-fns";
```

ESLint

[error] 'min' is defined but never used.

(@typescript-eslint/no-unused-vars)